xskillscore/core/utils.py

```diff
@@ -2,7 +2,37 @@
 import xarray as xr
 from xhistogram.xarray import histogram as xhist
 
-__all__ = ['histogram']
+__all__ = ['get_bin_edges', 'histogram']
+
+
+def get_bin_edges(bins):
+    """Returns bin edges of provided bins
```

Comment:
In hindsight, it's probably more robust to have users always specify bin_edges, rather than trying to infer them. Then we could remove this method altogether. Thoughts?

Reply:
yes. sounds more robust to have them applied by the user by default. or we add edges='infer' as another option? 
also wouldnt it be nice to have a keyword for standardized `[True, False]` forecasts? maybe this should all be covered in a separate issue/PR

Reply:
> also wouldnt it be nice to have a keyword for standardized `[True, False]` forecasts? maybe this should all be covered in a separate issue/PR

Sorry, I'm not sure what you mean by this?

Reply:
Also, I've removed `get_bin_edges`. Users now specify the edges rather than the centres.